features/src/main/feature/camel-features.xml

```diff
@@ -838,6 +838,7 @@
         <feature>scr</feature>
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <bundle dependency="true">mvn:org.ehcache/ehcache/${ehcache3-version}</bundle>
+        <bundle dependency="true">mvn:javax.cache/cache-api/1.1.0</bundle>
```

Comment:
Please use `${jcache-version}` instead, could you please also fix the hard-coded versions of cache-api elsewhere?

Reply:
Ok done